pkg/test/controller/reconcile.go

```diff
@@ -63,6 +66,28 @@ func StartTestManagerInstance(env *envtest.Environment, testType test.Type, whCf
 	return mgr, stopFunc
 }
 
+// Without the scheme, dynamic test returns `no kind is registered for the type
```

Comment:
Right - if you're not using unstructured, you need to be sure that the type is registered in the scheme.  As we want to read a particular field and we own the CRD, using types is a good plan.

Reply:
Moved the scheme registration to the test environment.